packages/fxa-settings/README.md

```diff
@@ -57,6 +59,45 @@ This application uses [Apollo client cache](https://www.apollographql.com/docs/r
 
 Access the client cache data in top-level objects via custom `use` hooks. At the time of writing, `useAccount` and `useSession` (see the "Sessions" section) will allow you to access `data.account` and `data.session` respectively inside components where that data is needed. See the "Testing" section of this doc for how to mock calls.
 
+### GQL error handling
+
```

Comment:
> Within `graphQLErrors`, the Apollo Server can return `AuthenticationError`, `ForbiddenError`, and `UserInputErrors`.

`UserInputError` should be singular and it looks like the docs do mention a generic `ApolloError`, maybe this:

"Apollo Server can throw `AuthenticationError`, `ForbiddenError`, `UserInputError`, or a generic `ApolloError` which Apollo Client contains in `graphQLErrors`."

Could also be helpful to link to https://www.apollographql.com/docs/apollo-server/data/errors/ and https://www.apollographql.com/docs/react/data/error-handling/

Reply:
Heh, I completely forgot to add what you had Slacked me. Apologies.